server/methods/accounts/accounts.js

```diff
@@ -59,6 +60,74 @@ export function verifyAccount(email, token) {
   return false;
 }
 
+
+/**
+ * updateEmailAddress
+ * @summary update a users email address
+ * @param {String} email - user email
+ * @returns {Boolean} - returns boolean
+ */
+export function updateEmailAddress(email) {
```

Comment:
After a bit of research, this seems to be the way people update email addresses on Meteor, first by adding a new one, then removing the old one. There is no callback on meteor's `Accounts.addEmail`, so I think you kind of have to send it back tot he client and do it this way.

Reply:
This seems sensible to me.